bedrock/firefox/templates/firefox/campaign/unfck/index.de.html

```diff
@@ -68,10 +68,10 @@
     {{ download_firefox(alt_copy='Download Firefox', download_location='primary cta') }}
 
     <div class="show-android mobile-badge hide-from-legacy-ie">
-      {{ google_play_button(href=android_url, id='play-store-link-primary') }}
+      {{ google_play_button(href=android_url, id='play-store-link-primary', extra_data_attributes={'data-download-location': 'primary cta'}) }}
     </div>
     <div class="show-ios mobile-badge hide-from-legacy-ie">
-      <a id="app-store-link-primary" href="{{ ios_url }}" data-link-type="download" data-download-os="iOS">
+      <a id="app-store-link-primary" href="{{ ios_url }}" data-link-type="download" data-download-os="iOS" data-download-location="primary cta">
```

Comment:
I have seen in other areas of bedrock adjust links having properties of `rel="external noopener"`, `data-display-name` and `data-download-version` ([example](https://github.com/mozilla/bedrock/blob/16b1010bde0aefd92bce3b16c4cab2258b110721/bedrock/firefox/templates/firefox/products/index.html#L76-L77)) Do we need that here as well?

Reply:
Good callout, will add 👍

Reply:
Ok, so thinking about this further I don't think we really need `noopener noreferrer` since these links don't open in a new tab. As for the GA attributes, very few app store badges on the site seem to use these so they might be optional (I checked both /mobile and /new).

Rather than start adding things to global macros based on some other examples, I think it's best we file an issue to help standardize things in a more systematic manner so we can also have some clearer docs.